package.json

```diff
@@ -4,21 +4,29 @@
   "description": "Hugo site for Histology Lab",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "watch-scss": "node-sass --output static/css static/scss && postcss -u autoprefixer -r static/css/main.css",
+    "build-scss": "nodemon -e scss -x 'npm run watch-scss'"
   },
   "repository": {
     "type": "git",
     "url": "git://github.com/ccnmtl/histologylab-static.git"
   },
-  "author": "",
+  "author": "Center for Teaching and Learning, Columbia University",
   "license": "BSD-2-Clause",
```

Comment:
I thought we use GPL3?

Reply:
It's up to whoever sets things up. I always use GPL3 - i think we have some libraries that are BSD or others.